### PR TITLE
yazawaj.com m_string fix

### PR DESCRIPTION
**Pull request by balestek**
Various small fixes:
- yazawaj : false positive
- 3Dnews : not working
- aahachat.org : not working
- apex.tracker.gg : javascript loading
- apteka.ee : closed
- ask.fm : not working
- community.avid.com : few headers needed to avoid beeing blocked
- behance : strings have changed

### CODE REVIEW

**Review by WebBreacher — CHANGES_REQUESTED** [review]

File: wmn-data.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
         "name" : "3DNews",
         "uri_check" : "https://forum.3dnews.tech/member.php?username={account}",
         "e_code" : 200,
-        "e_string" : "<title>Форум 3Dn Tech - Просмотр профиля:",
```
[the remark this answers is not in the document]
Comment (balestek) [reply wmn-data.json:28]: Hello,
The e_string in wmn-data.json is "<title>Форум **3Dn** Tech - Просмотр профиля:" while in the page source it's "Форум **3Dnews** Tech". As e_code and m_code are both 200, all checks fail.

File: wmn-data.json [hunk 2]
```diff
@@ -294,14 +294,25 @@
...
+            "User-Agent": "Mozilla/5.0 (Mozilla/5.0 (X11; Linux i686; rv:128.0) Gecko/20100101 Firefox/128.0",
+            "Accept-Language": "en-US,en;q=0.5",
+            "Origin": "https://apex.tracker.gg",
+            "Referer": "https://apex.tracker.gg/",
+            "TE": "trailers"
+        },
+        "e_code": 200,
+        "e_string": "platformInfo",
+        "m_code": 404,
+        "m_string": "message",
```
[the remark this answers is not in the document]
Comment (balestek) [reply wmn-data.json:310]: You are right

File: wmn-data.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (WebBreacher) [reply wmn-data.json:28]: I see you are correct. Thank you.

File: wmn-data.json [hunk 3]
```diff
@@ -428,7 +429,13 @@
         "uri_check" : "https://community.avid.com/members/{account}/default.aspx",
         "e_code" : 200,
         "e_string" : "My Activity",
-        "m_code" : 302,
```
[the remark this answers is not in the document]
Comment (balestek) [reply wmn-data.json:431]: You are right, I followed redirects, it's still 302 for m_code

File: wmn-data.json [hunk 4]
```diff
@@ -294,14 +294,25 @@
...
+            "Accept-Language": "en-US,en;q=0.5",
+            "Origin": "https://apex.tracker.gg",
+            "Referer": "https://apex.tracker.gg/",
+            "TE": "trailers"
+        },
+        "e_code": 200,
+        "e_string": "platformInfo",
+        "m_code": 404,
+        "m_string": "message",
+        "known": [
```
[the remark this answers is not in the document]
Comment (balestek) [reply wmn-data.json:311]: Do you want headers on a single line as well ?

**Comment by balestek** [comment]
I've updated the code according to your comments. Don't pay attention to Blogger and DockerHub, I cancelled the changes about them in the last commit.

File: wmn-data.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (WebBreacher) [reply wmn-data.json]: I don't think so due to wrapping. Sorry for the inconsistency but readability is important.

[the remark this answers is not in the document]
Comment (balestek) [reply wmn-data.json]: Ok, headers have been changed to 1 line (Apex Legends and Avid Community)

**Comment by WebBreacher** [comment]
Thanks for taking the time and effort to make these changes!!

**Pull request merged**